apps/mobile/src/state/chat/utils/createNewChatsFromFirstMessages.ts

```diff
@@ -40,6 +41,9 @@ export default function createNewChatsFromFirstMessages({
             showVexlbotInitialMessage: true,
             showVexlbotNotifications: true,
           },
+          tradeChecklist: {
+            ...emptyTradeChecklistInState,
```

Comment:
unnecessary spread? Not sure. maybe not. If the property is marked as readonly, it is ok not to spread, otherwise spread (just in case...)